Makefile

```
@@ -10,15 +10,18 @@ npm-update:
 	npm update
 
 # Building
-build-js:
+build-js: build-css
 	npm run dev
 
 build-js-production:
 	npm run build
 
-watch-js:
+watch-js: build-css
 	npm run watch
```

```suggestion
watch-js: build-css
	npm run watch &
	npm run sass:watch
```

Should make it possible to watch both at the same time

---

Mind though that I don't run the `npm run sass` commmand because it gives me errors here (see my message on Talk)

---

Ok, I've made the requested change